sample/src/main.c

```diff
@@ -1,19 +1,17 @@
-#include <stdio.h>
 #include <zephyr/kernel.h>
 #include <zephyr/logging/log.h>
 #include <zephyr/devicetree/gpio.h>
 #include <zephyr/drivers/gpio.h>
 #include <zephyr/device.h>
+
 #include <sid_api.h>
 #include <app_ble_config.h>
 #include <app_subGHz_config.h>
 #include <sidewalk_version.h>
-
-#include <sidewalk_callbacks.h>
 #include <pal_init.h>
 
-#define SID_WORK_Q_STACK_SIZE 8192
-#define SID_WORK_Q_PRIORITY 2
+#define SID_WORK_Q_STACK_SIZE CONFIG_SID_WORK_Q_STACK_SIZE
```

Comment:
much better, but the SID_WORK_Q_STACK_SIZE is used in only one place so the CONFIG_SID_WORK_Q_STACK_SIZE can be used directly instead,

the same with work_q_priority